src/sites/link/linkdrop.net.js

```diff
@@ -268,6 +264,13 @@
       o.observe(b, {
         attributes: true,
       });
+      
+      setTimeout(function(){
+        if(f.childNodes.length === 0 && !b.disabled){
+          _.info('clicking submit button, because recaptcha was still empty');
+          b.click();
+        }
+      }, 1000);
```

Comment:
```javascript
while (true) {
  await _.wait(1000);
  if (f.childNodes.length === 0 && !b.disabled) {
    _.info('clicking submit button, because recaptcha was still empty');
    b.click();
    break;
  }
}
```